src/tests/ftest/util/dmg_utils.py

```diff
@@ -21,32 +21,50 @@
     Any reproduction of computer software, computer software documentation, or
     portions thereof marked with this legend must also reproduce the markings.
     '''
+from __future__ import print_function
+
 import os
 import json
+from cmd_utils import Command, CommandParam
 
-class DmgFailed(Exception):
-    """Raise if DMG command or related activity doesn't work."""
+class DmgCmdUtils(Command):
+    """Defines a object representing a dmg (or daos_shell) command."""
 
-def get_dmg_script(dmgparams, testparams, basepath):
-    '''
-    Build a dmg command line
-    '''
+    def __init__(self, tool = "dmg"):
```

Comment:
(pylint-bad-whitespace) No space allowed around keyword argument assignment
(pylint-super-init-not-called) __init__ method from base class 'Command' is not called